src/components/Tabs/docs.md

```diff
@@ -0,0 +1,227 @@
+      isActive={state.activeChildIndex === 1}
+      onClick={() => {
+        setState({activeChildIndex: 1})
+      }}
+    />
+    <TabButton
+      text="Child Three"
+      isActive={state.activeChildIndex === 2}
+      onClick={() => {
+        setState({activeChildIndex: 2})
+      }}
+    />
+    <TabButton
+      text="Child Four"
+      isActive={state.activeChildIndex === 3}
+      onClick={() => {
+        setState({activeChildIndex: 3})
+      }}
+    />
+  </Scroller>
+</div>
+```
+
+## Generic Scroller with custom child nodes
+
+```react
+state: { activeChildIndex: 0 }
+---
+<div style={{ margin: '0 -20px' }} >
+  <Scroller
+    breakoutPadding={20}
+    activeChildIndex={state.activeChildIndex}
+  >
+    <button
+      {...plainButtonRule}
+      onClick={() => {
+        setState({activeChildIndex: 0})
+      }}
+    >
+      <FormatTag
+        color={state.activeChildIndex === 0 ? 'text' : 'textSoft'}
+        label='Alle'
+        count={3216}
+      />
+    </button>
+    <button
+      {...plainButtonRule}
+      onClick={() => {
+        setState({activeChildIndex: 1})
+      }}
+    >
+      <FormatTag
+        color={state.activeChildIndex === 1 ? 'text' : 'textSoft'}
+        label='Lob'
+        count={383}
+      />
+    </button>
+    <button
+      {...plainButtonRule}
+      onClick={() => {
+        setState({activeChildIndex: 2})
+      }}
+    >
+      <FormatTag
+        color={state.activeChildIndex === 2 ? 'text' : 'textSoft'}
+        label='Kritik'
+        count={608}
+      />
+    </button>
+    <button
+      {...plainButtonRule}
+      onClick={() => {
+        setState({activeChildIndex: 3})
+      }}
+    >
+      <FormatTag
+        color={state.activeChildIndex === 3 ? 'text' : 'textSoft'}
+        label='Wünsche'
+        count={1588}
+      />
+    </button>
+    <button
+      {...plainButtonRule}
+      onClick={() => {
+        setState({activeChildIndex: 4})
+      }}
+    >
+      <FormatTag
+        color={state.activeChildIndex === 4 ? 'text' : 'textSoft'}
+        label='Sonstiges'
+        count={637}
+      />
+    </button>
+  </Scroller>
+</div>
+```
+
+## Use TabButtons with next link
+
+```react
+state: { activeChildIndex: 0 }
+---
+<div style={{ margin: '0 -20px' }} >
+  <Scroller
+    breakoutPadding={20}
+    activeChildIndex={state.activeChildIndex}
+    center={true}
+  >
+    <Link href="/" passHref>
+      <TabButton
+        href="/"
+        text="This is a link"
+        isActive={state.activeChildIndex === 0}
+      />
+    </Link>
+  </Scroller>
+</div>
+```
+
+```code|lang-js
+import Link from 'next/link'
+state: { activeChildIndex: 0 }
+---
+<div style={{ margin: '0 -20px' }} >
+  <Scroller
+    breakoutPadding={20}
+    activeChildIndex={state.activeChildIndex}
+    center={true}
+  >
+    <Link href="/" passHref>
+      <TabButton
+        text="This is a link"
+        isActive={state.activeChildIndex === 0}
+      />
+    </Link>
+  </Scroller>
+</div>
+```
```

Comment:
<img width="932" alt="Screenshot 2021-11-17 at 13 45 57" src="https://user-images.githubusercontent.com/410211/142203295-dec7170f-c2f9-49af-98aa-9ff5e2e825e2.png">


```suggestion
```code|lang-js
<div style={{ margin: '0 -20px' }} >
  <Scroller
    breakoutPadding={20}
    activeChildIndex={0}
    center={true}
  >
    <Link href="/" passHref>
      <TabButton
        text="This is a link"
        isActive
      />
    </Link>
  </Scroller>
</div>
```
```